linuxNew/Jenkinsfile

```diff
@@ -0,0 +1,59 @@
+NODE_LABEL = "ci.role.test&&hw.arch.x86&&sw.os.linux&&ubuntu&&sw.tool.docker" // need to double check specific platform requirements for the package jobs, "fpm?"
+
+pipeline {
+    agent { 
+        label NODE_LABEL
+    }
+    stages {
+        stage('Set up') {
+            steps {
+                setup()
+            }
+        }
+        stage('Build and test package') {
+            steps {
+                dir('linuxNew') {
+                    buildAndTest()
+	            }
+            }
+        }
+        // For now looks like upload is combined with build and tests
+        stage('Upload') {
+            steps {
+                upLoadAritifact()
```

Comment:
minor name update: please change upLoadAritifact to uploadArtifacts

Reply:
Thanks! Updated.